### PR TITLE
V3

### LINKED ISSUE #34: Remove sub nav overrides
This would be a breaking change, but the code that is overwriting some styling is no longer necessary, once the internal brand is implemented across products #33

### PR DESCRIPTION
**Pull request by gvonkoss**
V3 of `o-header-services` does all of the things listed (and ticked off) below:

- [x] Include latest version of `o-brand` (v3)
- [x] Removes the option to use custom class names (as per https://github.com/Financial-Times/origami-proposals/issues/4)
- [x] Removes deprecated colours
- [x] Updates the URL for a chosen logo
- [ ] ~Adds search bar (?)~ (we've decided to postpone this until there is more concrete design)
- [x] Removes `o-header`
- [x] ~Removes drawer~
- [x] Reimplement drawer 😿 
- [x] Implements scrollable nav
- [x] Updates subnav
- [x] Improve accessibility
- [x] Updates SASS/JSDocs
- [x] Updates README + MIGRATION.md
- [x] Closes #34 
- [x] Make all mixins but `oHeaderServices` private (as per https://github.com/Financial-Times/origami-proposals/issues/6)
- [x] Removes H1
- [x] add support for whitelabel
- [x]  pivot and add drop downs to primary nav items
- [x] add brand customisation
- [x] update readme/migration again

### CODE REVIEW

**Comment by notlee** [comment]
Note for later, David pointed out we shouldn't have a `<h1>` in the header. 
https://github.com/Financial-Times/o-header-services/pull/51/files#diff-fbe1c4419c1b162c8686c84e1ca1d3e1R11

**Comment by JakeChampion** [comment]
Conflict in this PR on `src/scss/_drawer.scss`

File: src/js/drawer.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+			// display: none doesn't work with keyframes,
+			// so the element needs to be rendered before animated on open
+			setTimeout(() => this.nav.classList.add(this.class.open), 100);
+			this._toggleAriaAttributes(toggle);
+		} else {
+			this.nav.classList.remove(this.class.open);
+			this.burger.classList.remove('o-header-services__hambuger--open');
+			// display: none doesn't work with keyframes,
+			// so the element needs to be animated before hidden on close
+			setTimeout(() => this.nav.classList.add(this.class.hidden), 100);
```
Comment (notlee) [inline src/js/drawer.js]: Could the timeout be replaced with a listener for the [transitionend](https://developer.mozilla.org/en-US/docs/Web/Events/transitionend) or [animationend](https://developer.mozilla.org/en-US/docs/Web/Events/animationend) event?

Comment (gvonkoss) [reply src/js/drawer.js]: ohh good idea

**Pull request merged**